src/pages/search/project_search/ProjectSearch.tsx

```diff
@@ -40,7 +39,16 @@ export const ProjectSearch = ({ projectQuery }: ProjectSearchProps) => {
           ))}
         </List>
       ) : (
-        <Typography sx={{ mx: { xs: '0.5rem', md: 0 }, mt: '1rem' }}>{t('search.no_projects')}</Typography>
+        <NoSearchResults>
+          <Trans
+            i18nKey="no_search_results_list_default"
+            components={{
+              p: <Typography fontWeight="bold" />,
+              ul: <ul style={{ margin: 0, paddingLeft: '1.5rem' }} />,
+              li: <li />,
```

Comment:
Om vi finner ut at dette er noe som gjentas mange plasser etter hvert bør vi vurdere å dra det ut i en egen komponent eller samle det på annen måte. Spesielt for styling av ul tenker jeg. Men syns det gir mening i å ha det som det er nå så lite som det brukes nå 👍